docs/queries/luke-query/result-of-a-luke-query/document.md

```diff
@@ -26,7 +26,7 @@ This provides detailed information on the fields and their values, including
 
 ### Solr document
 
-A [read-only document](/documents/#read-only-document). Can be retrieved with
+A [read-only document](../../../documents.md#read-only-document). Can be retrieved with
 `$docInfo->getSolr()`.
```

Comment:
```suggestion
A [read-only document](/docs/documents.md#read-only-document).
Can be retrieved with `$docInfo->getSolr()`.
```

WDYT? it would simplify the mental load of checking how many folders back the link is pointing to :)